azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,43 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps["iot edge hierarchy"] = """
+        type: group
+        short-summary: Commands to deploy and manage nested IoT Edge device hierarchies.
+    """
+
+    helps["iot edge hierarchy create"] = """
+        type: command
+        short-summary: |
+          Deploy nested edge hierarchies to an IoT Hub.
+          This can be done with inline device arguments or by supplying a nested edge configuration file in YAML or JSON format.
+          Review examples and parameter descriptions for details on how to fully utilize this operation.
+
+        examples:
+        - name: Delete all existing devices in the Hub, and create a simple nested edge device hierarchy
+            containing 2 parent devices with 1 child device each. Also specifies module deployments
+            to configure the devices and visualizes the operations as they progress.
+          text: |
+            az iot edge hierarchy create -n {hub_name} --clean --visualize
```

Comment:
General question, currently the device bundle output is always produced right? If so, should there be an opt-in way to activate, or a way to deactivate? 

Scenario I'm thinking - what if a user wants to create or update a set of edge device identities but don't need the device side setup scripts?

Reply:
I agree, this was something that I realized as well - we should either:
- Specify that the device bundle output always happens unless a `--no-output`-esque param is provided
- Only create device bundle output if a path is provided.